app/views/users/noticed_notifications/index.html.erb

```diff
@@ -44,6 +44,10 @@
           <h6><%= t("users.notifications.no_notifications") %></h6>
         </div>
       </div>
+    <% else %>
+      <div class="container pagination-cont">
+        <%= will_paginate @notifications, renderer: NotificationPaginateRenderer %>
+      </div>
```

Comment:
We need to add this for `@unread` as well.
Because if you see ,unread tab is not rendering pagination div.

Reply:
Ohk cool. I'll add it in unread tab as well

Reply:
But one thing still remains, when we click on the unread pagination number, we are getting to the new page with all notifications instead of the unread tab.

Reply:
Yes, we need to make query paramaters, something like this for all: `/notifications?page=1&filter=all` and unread: `/notifications?page=1&filter=unread`

@tachyons

Reply:
Do not worry about that in this PR, we can make a followup issue on that!